### ISSUE TITLE
Email: Siteimprove Comments on WCAG 3.0 Draft (2 - Testing is difficult to carry out efficiently)

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/wcag3#260]
[Comment from email:](https://lists.w3.org/Archives/Public/public-silver/2021Apr/0031.html)
Testing is difficult to carry out efficiently
In WCAG 3.0, one can no longer simply report on issues that fail WCAG. Instead, one has to report the degree to which all outcomes, including those part of successful criteria, are met. This is because the test result is based on an overall score of all criteria (with some caveats), not on finding out which criteria pass or fail or which are relevant in the testing context. Given that there are also no longer different levels of conformance (A, AA, AAA), one must score each criteria and all its outcomes for every tested process. What is more, the scoring formulae for these outcomes vary. Given, then, the vast number of the outcomes, these formulae are perforce very difficult to memorize reliably and must be consulted carefully during a test.
Conclusion: The testing procedure becomes exceedingly taxing, as each of the (hundreds?)  of individual outcomes must be checked for their specific success formulae and the respective data collected and calculated for the test artefact. It goes without saying that the manual effort required is vast and altogether unsuitable for current manual testing approaches whereby the auditor can report only on criteria that fail, and by this means invest time and effort on explaining how and why the criterial fail and how the service owner can best amend the issues in order to improve the service accessibility. 
Now, however, the focus in WCAG 3.0 shifts from testing that serves the customer and the web service users to a testing that mechanically, yet unproductively, reports on passed atomic outcomes, with a lot of room for calculation and data assembly errors yet with little benefit for the service owner or its users.
Today, a manual WCAG test report of a web service can comprise 20 to 30 pages. If the report expands manifold owing to the overhead discussed above, one will have to re-think the entire testing procedure to somehow accommodate for all the mandatory data. 
This will likely preclude many manual auditors from offering their services in the future, as, on one hand, they cannot simply absorb this level of extra work without raising their prices substantially, and, on the other, the customers who here in Europe are mainly composed of the public sector cannot, in their turn, pay more for these services, which are already pricey due to the degree of manual work required. It is likely that the industry will thus shift towards increasingly automated and hence poorer quality testing procedures that conform to WCAG in name only. One gathers that is not a desirable outcome.

This, one notes, is in stark contrast to the intent of WCAG 3.0, which is to augment a more qualitative and relevant testing approach. It will now likely do the opposite.
[WCAG 3 comments_Siteimprove_Codebusters_April7_2021.docx](https://github.com/w3c/silver/files/6281581/WCAG.3.comments_Siteimprove_Codebusters_April7_2021.docx)

**Comment by hidde** [comment]
The issues SiteImprove discussed are related to the first public working draft of WCAG 3 and are no longer relevant to the [latest draft that was published 10 September](https://www.w3.org/TR/wcag-3.0/). The scoring mechanism that is of concern is no longer part of this latest draft, and the amount of requirement has changed.

Closing this issue as no longer relevant, feedback on the latest draft is of course welcomed.
